### ISSUE TITLE
Firefox 105 support

### ISSUE THREAD

**Issue by lukewarlow** [twilio/twilio-video-processors.js#49]
**What is the feature that you would like to see? Please describe.**
Firefox 105 which is due for release September 20th comes with Offscreen Canvas support. I believe I'm correct in thinking that's the last API required from Firefox for support. If so it would be good if the browser supported check could be updated to account for this.

**Is your feature request related to a problem? Please describe.**
Being able to offer virtual background functionality to Firefox users as well as Chromium would be good.

**Comment by Dosant** [comment]
Thumbs up!

I've tried to test this out in firefox with the recent twilio video js version (2.24.x), by just [removing the browser ](https://github.com/twilio/twilio-video-processors.js/blob/c9f8da5c10cdd38b4e22cdbb3ef810cc6c39e148/lib/utils/support.ts#L6) check didn't work. 

I got the `ns_error_not_initialized` initialized error [here](https://github.com/twilio/twilio-video.js/blob/5d4d454fba88af9a880b9c678fa59717e9e8bd42/lib/media/track/videotrack.js#L293). I found [this thread ](https://stackoverflow.com/questions/74048056/capturestream-from-transferred-canvas-not-working-in-firefox) where was suggested to call `canvas.getContext('2d');` before starting capturing the stream. This fixed the error, but then, in my case, I just got the frozen frame (with a properly applied background filter) and the video wasn't running. 

----

I then was wondering why [the demo](https://twilio.github.io/twilio-video-processors.js/examples/virtualbackground/) and realized that it was using 2.0.0-rc1 that I didn't know to exist.  
Seems like there was no activity for a while ;(

Dear Twilio team, could you please let us know the status of 2.0.0-rc1 branch? Is there any chance it will be finished or was the project abandoned?

**Comment by ravkoopharma** [comment]
Recently we have implemented Virtual Background feature in our Web App, it's working great with Google Chrome, Brave Browser but it's not working with Firefox. Our front end is developed with Angular 11 we are using twilio-video - ^2.26.1 version and video-processors - ^1.0.2. AS @Dosant mentioned [the demo](https://twilio.github.io/twilio-video-processors.js/examples/virtualbackground/) working fine in Firefox.

Can anyone help me with that is there anything we are missing. We have followed same instruction and code which is mentioned in demo and tutorials.

Thanks in Advance.
